### PR TITLE
Fix commands message

### PR DESCRIPTION
**Pull request by Douile**
Replace hq, stats with hq, bot info

### CODE REVIEW

**Comment by Chew** [comment]
The commands are `hq, bot` and `hq, info`, they're separate commands, just FYI :D

**Comment by Douile** [comment]
rly?

**Comment by Douile** [comment]
k

**Comment by Douile** [comment]
I don't get ruby

**Comment by Chew** [comment]
You just had to remove info or bot... lol

File: lib/plugins/about.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ module About
           '`hq, commands` - Find bot commands',
           '`hq, ping` - Ping the bot',
           '`hq, invite` - Invite the bot',
-          '`hq, stats` - Find stats on the bot',
+          '`hq, bot info` - Find stats on the bot',
```
Comment (Chew) [inline lib/plugins/about.rb:32]: Remove "bot" or "info" to fix :)

Comment (Douile) [reply lib/plugins/about.rb:32]: IK I meant I didn't get the
%I[bot info]

Comment (Douile) [reply lib/plugins/about.rb:32]: https://github.com/Chewsterchew/HQ-DiscordBot/blob/71cb6cf84a36a5941d209b434219cd0a33f7e07b/lib/plugins/about.rb#L66

Comment (Chew) [reply lib/plugins/about.rb:32]: That's a shorthand for an array of hashnames, essentially the same as [:help, :bob] or something

Comment (Douile) [reply lib/plugins/about.rb:32]: Ah, okay.

Comment (Douile) [reply lib/plugins/about.rb:32]: https://github.com/Chewsterchew/HQ-DiscordBot/pull/2

Comment (Chew) [reply lib/plugins/about.rb:32]: Why'd you make a new one, just edit this one.

Comment (Douile) [reply lib/plugins/about.rb:32]: Didn't want to press the scary **RE-OPEN** button

**Pull request closed without merging**